distribution/docs/src/main/resources/content/_managing/_configuring/configuring-ftp-endpoint.adoc

```diff
@@ -0,0 +1,27 @@
+:title: Configuring FTP Endpoint
+:type: subConfiguration
+:status: published
+:parent: Configuring Endpoints
+:summary: Configuring FTP Endpoint
+:order: 02
+
+== {title}
+
+The FTP endpoint provides a method for ingesting files directly into the ${branding} Catalog using the FTP protocol
+The files sent over FTP are not first written to the file system, as the Directory Monitor does, but instead the FTP stream of the file is ingested directly into the ${branding} catalog, thus avoiding extra I/O overhead.
```

Comment:
✏️ Could be useful to link `Content Directory Monitor` to the the corresponding docs.

Reply:
added.